huaweicloud/services/lts/resource_huaweicloud_lts_cross_account_access.go

```diff
@@ -0,0 +1,292 @@
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"log_stream_name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"log_group_id": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"log_group_name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"agency_project_id": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"agency_name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"agency_domain_name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"tags": common.TagsSchema(),
+			"created_at": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"access_config_type": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+		},
+	}
+}
+
+func resourceCrossAccountAccessCreate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	cfg := meta.(*config.Config)
+	region := cfg.GetRegion(d)
+
+	var (
+		createHttpUrl = "v2.0/{project_id}/lts/createAgencyAccess"
+		product       = "lts"
+	)
+	ltsClient, err := cfg.NewServiceClient(product, region)
+	if err != nil {
+		return diag.Errorf("error creating LTS client: %s", err)
+	}
+
+	createPath := ltsClient.Endpoint + createHttpUrl
+	createPath = strings.ReplaceAll(createPath, "{project_id}", ltsClient.ProjectID)
+
+	createOpt := golangsdk.RequestOpts{
+		KeepResponseBody: true,
+	}
+	createOpt.JSONBody = utils.RemoveNil(buildCreateBodyParams(d, ltsClient.ProjectID))
+	createResp, err := ltsClient.Request("POST", createPath, &createOpt)
+	if err != nil {
+		return diag.Errorf("error creating LTS cross account access: %s", err)
+	}
+
+	createRespBody, err := utils.FlattenResponse(createResp)
+	if err != nil {
+		return diag.FromErr(err)
+	}
+
+	id, err := jmespath.Search("[0].access_config_id", createRespBody)
+	if err != nil {
+		return diag.Errorf("error creating LTS cross account access: ID is not found in API response")
+	}
+	d.SetId(id.(string))
+
+	// deal tags
+	if tags, ok := d.GetOk("tags"); ok {
+		createTagsHttpUrl := "v1/{project_id}/{resource_type}/{resource_id}/tags/action"
+		createTagsPath := ltsClient.Endpoint + createTagsHttpUrl
+		createTagsPath = strings.ReplaceAll(createTagsPath, "{project_id}", ltsClient.ProjectID)
+		createTagsPath = strings.ReplaceAll(createTagsPath, "{resource_type}", "ltsAccessConfig")
+		createTagsPath = strings.ReplaceAll(createTagsPath, "{resource_id}", d.Id())
+
+		createTagsOpt := golangsdk.RequestOpts{
+			KeepResponseBody: true,
+		}
+		createTagsOpt.JSONBody = utils.RemoveNil(buildTagsBodyParams(tags.(map[string]interface{}), "create"))
+		_, err := ltsClient.Request("POST", createTagsPath, &createTagsOpt)
+
+		if err != nil {
+			return diag.Errorf("error creating LTS cross account access tags: %s", err)
+		}
```

Comment:
please using public function (`utils.UpdateResourceTags`) to batch create or delete the tags.

Reply:
the public function is not support service type of LTS, and the API is not work as other tags API.